includes/class-wc-payments-db.php

```diff
@@ -38,28 +38,26 @@ public function order_from_charge_id( $charge_id ) {
 	 * @return array[]
```

Comment:
Should we be more specific about the return type here for easier reference? I feel it helps with testing.

Reply:
Discovered that is pretty complex to represent an array of objects with mixed types in the phpdoc directly, something for a wider discussion later, resolving this for now.